Added sponsor resources section and fauna

kdehaan opened the pull request.

kdehaan committed:
Added sponsor resources section and fauna

MarkBennett commented on index.html:
```diff
@@ -265,6 +265,45 @@ <h3>FindCollabs</h3>
           </p>
         </section>
       </div>
+      <section>
```
It's come up that our front page is getting too full, rather than add this there I started a `/resources.md` page but haven't linked it yet as there's not enough content.

Do you want to add this there instead, then we can link up from the home page?

ajyong replied:
We also have Digital Ocean providing credits for COVID-19-related projects, IIRC.  I support having a `/resources.md` started and putting this and DO's resources there.

kdehaan replied:
I appreciate that markdown is being used to make things easier, but I'm familiar enough with HTML that it's getting in the way of some of the formatting I would like. Regardless, I took a stab at it and I think it's okay, though I would like to use the same buttons as on the main site if at all possible.

MarkBennett commented on index.html:
```diff
@@ -265,6 +265,45 @@ <h3>FindCollabs</h3>
+        </p>
+        <h3>Fauna</h3>
+        <p>
+          <a href="https://fauna.com/"> FaunaDB</a> is a serverless cloud
```
On other websites, I've noticed they explain if the offer is available globally, or just in certain regions. I assume that this doesn't apply here given the nature of the service but can you specify that there are no restrictions so we don't get questions?

kdehaan replied:
I added some more info from Fauna, should be _very_ clear whats being offered now

ajyong replied:
I've checked this project out locally, and I do agree that Markdown may not serve us and participants well.  As a participant, I just want the tl;dr, and right now this doesn't scale with just 1 resource:

<img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208795-8712e180-7462-11ea-862f-636fa7ee0e84.png">

It's a bit of a learning curve, and at the risk of making this PR take way too long to get shipped, could we consider making a Resources  index template in HTML and authoring each individual resource as a Jekyll [Collection item](https://jekyllrb.com/docs/step-by-step/09-collections/)?  I believe this will allow for:

1. The visual freedom to get something akin to [Hyperspace](https://html5up.net/uploads/demos/hyperspace/index.html)'s image+summary listview: 
  <img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208953-f2f54a00-7462-11ea-88d1-bc1f09d787b2.png">
2. Allow public contributors to add another Jekyll resource via PR, without needing to worry about HTML.

MarkBennett reviewed (changes requested):
Thanks for putting this together @kdehaan!

Do you mind moving this content to the `/rsources.md` page instead? Also, can you specify what if any restrictions apply to using FaunaDB?

ajyong commented on index.html:
```diff
@@ -265,6 +265,45 @@ <h3>FindCollabs</h3>
         </section>
       </div>
+      <section>
+        <h2>Donated Services</h2>
```
Food for thought with this new Resources page: we could spotlight Sponsor resources as the topmost section and have anything else further down.

kdehaan committed:
Merge remote-tracking branch 'origin/master' into add-sponsor-resources

kdehaan committed:
Transferred fauna to resources.md and added a link to the page from the main site

ajyong reviewed (changes requested):
Oops, I thought I submitted my review yesterday.  These are a bit old but still relevant if we decide to go with an HTML/Jekyll collection solution.

ajyong commented:
(Pasting outside of my outdated review, deleted it from its old spot)

I've checked this project out locally, and I do agree that Markdown may not serve us and participants well.  As a participant, I just want the tl;dr to grab my attention, and right now this is too heavy at first glance:

<img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208795-8712e180-7462-11ea-862f-636fa7ee0e84.png">

It's a bit of a learning curve (and risks making this PR take way too long to get shipped), but could we consider making a Resources  index template in HTML and authoring each individual resource as a Jekyll [Collection item](https://jekyllrb.com/docs/step-by-step/09-collections/)?  I believe this will allow for:

1. The visual freedom to get something akin to [Hyperspace](https://html5up.net/uploads/demos/hyperspace/index.html)'s image+summary listview: 
    <img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208953-f2f54a00-7462-11ea-88d1-bc1f09d787b2.png">
1. Allow public contributors to add another Jekyll resource via PR, without needing to worry about HTML.
---
Hyperspace also has an [Elements](https://html5up.net/uploads/demos/hyperspace/elements.html) page that shows the full capabilities of this template.  I've used Material Design's concept of [Cards](https://material.io/components/cards/) in similar use cases and think they would be perfect for a Resource index page, but I'm unsure if that's possible with the existing Hyperspace styles.

#### Card Example
<img width="776" alt="image" src="https://user-images.githubusercontent.com/6334517/78209519-76636b00-7464-11ea-97ed-dfe491e2c776.png">

kdehaan commented:
@ajyong I don't have the time right now to iterate on this. Requesting either a) you take over or b) we get _something_ out and tweak it later

ajyong reviewed (approved):
Let's merge this, I'll copy my comment into a Clubhouse story to iterate on it.  Thanks for contributing @kdehaan!

MarkBennett reviewed (approved):
Changes look good to me @kdehaan. Ship it, and we can add sexy up the resources page later.

MarkBennett commented:
@ajyong you're right that we can use collections to do some slicker layouts.

For now, let's ship this as is and get up an MVP then iterate from there.
